### PR TITLE
A0-2230, A0-2520: visual corrections after review

### PR DESCRIPTION
**Pull request by youPickItUp**
Those two issues were connected

### CODE REVIEW

File: packages/extension-ui/src/components/AccountNamePasswordCreation.tsx [hunk 1]
```diff
@@ -180,44 +168,50 @@ function AccountNamePasswordCreation({
...
+`;
+
+const StyledAddress = styled(Address)`
+  margin-bottom: 24px;
+`;
 
-  .spacer {
-    height: 16px;
+const InputsWrapper = styled.div`
+  & > * {
```
Comment (jalooc) [inline packages/extension-ui/src/components/AccountNamePasswordCreation.tsx:184]: Isn't
```suggestion
  & > *:not(:last-child) {
```
needed?

File: packages/extension-ui/src/components/Dropdown.tsx [hunk 2]
```diff
@@ -57,7 +57,7 @@ function Dropdown({
   return (
     <div className={className}>
       <Label
-        $active
+        $small
```
Comment (jalooc) [inline packages/extension-ui/src/components/Dropdown.tsx:60]: Does this have to be a transcient prop? If I see correctly, the underlying props are not passed-through to a DOM element, so I think this is not necessary?

Reasoning: apart of slight ick when starting a prop with `$`, imho differentiating transient props from regular props somewhat limits the solidity of a component by exposing internal mechanisms. Not a biggie of course, just a _polishing_ discussion.

**Review by jalooc — APPROVED** [review]
Lgtm, just a couple of non-blocking comments.

File: packages/extension-ui/src/Popup/Derive/SelectParent.tsx [hunk 3]
```diff
@@ -167,36 +155,36 @@ function SelectParent({
...
+        </AddressWrapper>
+        <InputWrapper ref={passwordInputRef}>
           <ValidatedInput
             component={InputWithLabel}
             data-input-password
             label={t<string>('Main account password')}
             onValidatedChange={_onParentPasswordEnter}
             type='password'
-            validator={isPasswordValid}
+            validator={Result.ok}
```
[the remark this answers is not in the document]
Comment (youPickItUp) [reply packages/extension-ui/src/Popup/Derive/SelectParent.tsx:187]: it makes a lot of sense, but I would leave it for now if you don't mind. Inputs in signer require refactor anyway :sweat_smile:

File: packages/extension-ui/src/components/AccountNamePasswordCreation.tsx [hunk 4]
```diff
@@ -5,58 +5,44 @@ import React, { useCallback, useContext, useState } from 'react';
...
+  parentName?: string | null;
+} & (
+  | {
+      setGenesis: (newGenesisHash: string) => void;
+      isDeriving?: false;
+    }
+  | {
+      setGenesis?: never;
+      isDeriving: true;
+    }
```
[the remark this answers is not in the document]
Comment (youPickItUp) [reply packages/extension-ui/src/components/AccountNamePasswordCreation.tsx:44]: I think what happened is - I refactored styles, which fixed typing and revealed error :sweat_smile: 
I fixed the typing without going into further refactor.
if you don't mind I would leave it

File: packages/extension-ui/src/components/AccountNamePasswordCreation.tsx (hunk 1 above)
Comment (youPickItUp) [reply packages/extension-ui/src/components/AccountNamePasswordCreation.tsx:184]: I think I want margin on the last child as well, to space properly from footer

**Pull request merged**